src/main/java/page_objects/IndexPage.java

```diff
@@ -0,0 +1,23 @@
+package page_objects;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class IndexPage extends PageObject{
+
+    public IndexPage(WebDriver driver){
+        super(driver);
+    }
+
```

Comment:
Login link can be moved to TopMenuPage object. It is part of index but it is displayed on every page, so please move that to it's object